.github/workflows/weekly-release.yaml

```
@@ -135,11 +162,8 @@ jobs:
           ssh-private-key: ${{ secrets.MATCH_GIT_PRIVATE_KEY }}
       - name: Build and Release to TestFlight
         env:
-          MATCH_GIT_URL: ${{ secrets.MATCH_GIT_URL }}
-          MATCH_PASSWORD: ${{ secrets.FASTLANE_MATCH_PASSWORD }}
           APPSTORE_KEY_ID: ${{ secrets.APPSTORE_KEY_ID }}
-          APPSTORE_ISSUER_ID: ${{ secrets.APPSTORE_ISSUER_ID }}
-          APPSTORE_PRIVATE_KEY: ${{ secrets.APPSTORE_PRIVATE_KEY }}
+          APPSTORE_ISSUER_ID: ${{ secrets.APPSTORE_ISSUER_ID }
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end marker }}, expecting '}'

(expression)